stubs/Markdown/markdown/extensions/codehilite.pyi

```diff
@@ -18,9 +18,11 @@ class CodeHilite:
     tab_length: Any
     hl_lines: Any
     use_pygments: Any
+    options: Dict[str, Any]
     def __init__(
         self,
         src: Optional[Any] = ...,
+        *,
         linenums: Optional[Any] = ...,
         guess_lang: bool = ...,
         css_class: str = ...,
```

Comment:
It's still missing `**options: Any`, as you can pass in any options you want (and they go to `self.options`).

Reply:
Why do we want to add `**options: Any` when all the possible arguments that will be passed through `**options` are already defined here? :thinking: 
If there are some custom arguments that we want to pass, how will they be used inside this method?

Reply:
I've added the commit with this change.
Still, I'd be happy if you could answer the above question. :)

Reply:
Read the source code. It pops **some** things from the `options` dict, but everything that it doesn't pop goes to `self.options`.

Reply:
Okay, we can use the custom arguments using `self.options`.